src/OpenLoco/Vehicles/VehicleHead.cpp

```diff
@@ -3277,4 +3278,18 @@ namespace OpenLoco::Vehicles
         regs.esi = reinterpret_cast<uint32_t>(this);
         call(0x004B08DD, regs);
     }
+
+    // 0x004C3BA6
+    currency32_t VehicleHead::calculateRunningCost() const
+    {
+        currency32_t totalRunCost = 0;
+        const Vehicle train(this);
+        for (const auto& car : train.cars)
+        {
+            auto* vehObj = ObjectManager::get<VehicleObject>(car.front->object_id);
+            currency32_t runCost = (vehObj->run_cost_factor * currencyMultiplicationFactor[vehObj->run_cost_index]) / 1024;
```

Comment:
We're probably going to be using `currencyMultiplicationFactor` like this more often. Worth having a function to do this in Economy.cpp? I realise you moved this code, so perhaps outside of the scope of this PR.

Reply:
Will do a follow on PR.